docs/develop/api-reference/index.mdx

```diff
@@ -194,6 +198,8 @@ puts client.caches.to_a.join(", ")
 
 ## Data APIs
 
+These API methods are used to interact with data (DML) in caches.
```

Comment:
what's DML?

Reply:
Data modification language. I forgot to put that in and now that you call it out, I will just remove it. it doesn't make sense now.